python/taichi/profiler/kernelprofiler.py

```diff
@@ -28,8 +35,11 @@ def insert_record(self, time):
         self.max_time = max(self.max_time, time)
 
 
-class Profiler:
-
+class KernelProfiler:
+    """Acquires kernel profiling records from a backend,
```

Comment:
This is the official docstring style: https://www.python.org/dev/peps/pep-0257/#id17

Use a one-line summary first, followed by an empty line, followed by however many number of lines necessary